cluster-autoscaler/provisioningrequest/provreqclient/testutils.go

```diff
@@ -35,7 +36,7 @@ import (
 )
 
 // NewFakeProvisioningRequestClient mock ProvisioningRequestClient for tests.
-func NewFakeProvisioningRequestClient(ctx context.Context, t *testing.T, prs ...*provreqwrapper.ProvisioningRequest) *ProvisioningRequestClient {
+func NewFakeProvisioningRequestClient(ctx context.Context, t *testing.T, prs ...*provreqwrapper.ProvisioningRequest) (*ProvisioningRequestClient, *FakeProvisioningRequestForceClient) {
```

Comment:
Why have one function to return two clients - one of which is then ignored, instead of having a dedicated function to create either cached or cache-less client?